rfcs/0000-community-coordination-hub.md

```diff
@@ -0,0 +1,150 @@
+---
+feature: community_coordination_hub
+start-date: 2020-07-30
+author: David Arnold <dar@xoe.solutions>
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Adopt goals, spirit, concept and execution of [`kubernetes/community`]
+where it _does_ fit [us](https://nixos.org/).
+
+[`kubernetes/community`]: https://github.com/kubernetes/community
+
+# Motivation
+[motivation]: #motivation
+
+We lack an optimum level of _empowerment_ on distinct vital aspects for
+the best [leverage](https://nav.al/least) on the available resources:
+
+- Efficient Management Processes, that bias the whole community towards 
+  _coordinated and optimal Resource Employment_.
+- Organizational Structure, that enacts necesary degrees of _Division of Labor_ 
+  and _Coordination_.
+- Organizational Culture, that helps to overcome the complexity bias. ([KISS](https://en.wikipedia.org/wiki/KISS_principle))
+
+Analysis inspired by [New St.Gallen Managament Model].
+
+[New St.Gallen Managament Model]: https://cio-wiki.org/wiki/The_New_St._Gallen_Management_Model
```

Comment:
Maybe it's not relevant to the PR? With all due respect for (whatever that is :)) the "New St.Gallen Managament Model", it's great that you give credit, but it could be mentioned in your PR's initial comment perhaps? Or our discourse thread? I'm not strongly opinionated but it's a sort of distraction IMO, just thinking out loud.

Reply:
I think you are right, it's not relevant at least in this section.

Created a Credits / School of Thoughts section at the bottom. I kind of like to encourage reader to dive into more deeply, yet as an afterthought.